core/build.sbt

```diff
@@ -138,6 +138,18 @@ lazy val akkastream =
             AkkaShardingTyped,
             AkkaCluster,
             AkkaManagement,
+            // Persistence
+            AkkaPersitenceTyped,
+            AkkaSerialization,
+            AkkaPersistenceQuery,
+            AkkaPersistenceJDBC,
+            AkkaPersistenceCassandra,
+            // DB drivers
```

Comment:
It would be great if the user could choose which to use through (some help of the) sbt plugin as well.

Reply:
Here really is a choice between AkkaPersistenceJDBC and appropriate driver and AkkaPersistenceCassandra. The advantage of including both here allows user to experiment with different persistence implementations purely through configuration without rebuild an app. I do like adding all of them, not too much overhead